client/types/disputes.d.ts

```diff
@@ -64,6 +64,24 @@ export interface Dispute {
 	balance_transactions?: BalanceTransaction[];
 }
 
+export interface CachedDispute {
+	wcpay_disputes_cache_id: number;
+	stripe_account_id: string;
+	dispute_id: string;
+	charge_id?: string;
+	amount: number;
+	currency: string;
+	reason: DisputeReason;
+	source?: string;
+	order_number?: number;
+	customer_name?: string;
+	customer_email?: string;
+	customer_country?: string;
```

Comment:
Doesn't this syntax mean that `charge_id`, `source`, etc keys can be absent as not set? In reality, I think server could return empty value, but the key still should be there. 

Or could be I'm misunderstanding something in TS syntax...

Reply:
Updated!